git-sync.php

```diff
@@ -107,12 +107,92 @@ public function init()
         $this->git = $this->controller->git;
     }
 
+    /**
+     * Execute command in background, cross-platform
+     * @param $cmd string Command to execute
+     * @return int|string Command result
+     */
+    public static function execInBackground($cmd)
+    {
+        if (false !== stripos(PHP_OS, 'win')) {
+            // Windows platform
+            return pclose(popen('start /B ' . $cmd, 'r'));
+        }
+        // Unix-like platform
+        return exec($cmd . ' > /dev/null &');
+    }
+
+    /**
+     * Checks if /bin/plugin exists in the given path
+     * @param string $path Path to check
+     * @return boolean
+     */
+    private static function doesBinPluginExist($path)
+    {
+        // Find /bin path
+        if (!($binPath = \realpath($path . DIRECTORY_SEPARATOR . 'bin'))) {
+            // /bin path could not be found automatically
+            return false;
+        }
+
+        // Find /plugin executable
+        $pluginPath = $binPath . DIRECTORY_SEPARATOR . 'plugin';
+        return \file_exists($pluginPath) && \is_readable($pluginPath);
```

Comment:
You can also add:
```php
&& \is_executable($pluginPath)
```

Reply:
That's what I did first, but on Windows it returns `false`, and yet it works!

![image](https://user-images.githubusercontent.com/3274103/39370824-323e9148-4a40-11e8-8436-b6c345a1be67.png)

Don't ask me why.

Reply:
👎  Please uninstall windobe :P